app/services/sip_arrange.service.js

```diff
@@ -59,36 +60,13 @@ factory('SipArrange', ['Restangular', function(Restangular) {
       });
     };
 
-    var format_entries = data => {
-      return data.entries.map(element => {
-        var child = {
-          title: element,
-          path: parent ? parent.path + '/' + element : element,
```

Comment:
In browse-helpers, if the parent_path is null, then '/' is prepended to element.  Does this matter?